spec/book_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  it do
```

Comment:
We can put each it in just one line :)
` it { should validate_presence_of(:genre) }`

We can also use is_expected.to:
`it { is_expected.to validate_presence_of(:genre) }`

Reply:
https://github.com/thoughtbot/shoulda-matchers#should-vs-is_expectedto